doc/advanced-configuration/outgoing-connections.md

```diff
@@ -205,13 +205,18 @@ Below is a sample configuration:
 
 `ConnectionOptions` can take the following `{key, value}` pairs:
 
-* `{server, HostName}` - string, default: `"http://localhost"` - the URL of the destination HTTP server (including a port number if needed).
+* `{http_client, Client}` - `gun` or `fusco`, defaults to `fusco`.
```

Comment:
Why do we even keep `fusco` if `gun` supports both HTTP 1.1 and HTTP/2? Isn't that just redundant code? I know that `fusco` can keep connection alive for however long we want, but doesn't `gun` do the same?